<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1640</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(teste.jpg);</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_login">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>380</y>
      <width>461</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QFrame{
background:#333;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Push_Logar">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>320</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;
background:red;
font-size:20px;
font-family:century gothic;
color:white;
</string>
     </property>
     <property name="text">
      <string>Logar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Login_usuario">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color:white;
font-size:20px;
border-bottom:2px solid #717072;
}
QLineEdit:hover{


	border-bottom:2px solid rgb(212, 212, 212);
}
QLineEdit:pressed{


	border-bottom:2px solid rgb(212, 212, 212);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>11</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>CPF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Push_Voltar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:#333;
border:3px;
font-size:13px;
font-family:century gothic;
color:white;
}

QPushButton:hover {
	border:1px solid rgb(93, 93, 93)

}</string>
     </property>
     <property name="text">
      <string>VOLTAR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Push_Registrar">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>380</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:#333;
border:3px;
font-size:13px;
font-family:century gothic;
color:white;
}

QPushButton:hover {
	border:1px solid rgb(93, 93, 93)

}</string>
     </property>
     <property name="text">
      <string>REGISTRAR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Login_senha">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color:white;
font-size:20px;
border-bottom:2px solid #717072;
}
QLineEdit:hover{


	border-bottom:2px solid rgb(212, 212, 212);
}
QLineEdit:pressed{


	border-bottom:2px solid rgb(212, 212, 212);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>DATA DE NASCIMENTO</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_login_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>290</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QFrame{
background:#333;

border-bottom:5px solid white;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Text_Login">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>28</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

color:white;
border:none;
 font-size:22px;
 font-family:century gothic;
}</string>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
